onnxruntime/core/framework/TensorSeq.h

```diff
@@ -28,6 +28,9 @@ class TensorSeq {
   }
 
   void SetElements(std::vector<Tensor>&& tensors) {
+    // TODO:
+    // (1) Do we want to ensure elem_type is set before accepting tensors ?
```

Comment:
I think the caller ensures this. We can instead document this function accordingly.

Reply:
Documented in a comment